packages/bot-web-ui/src/components/dashboard/dashboard-component/__tests__/dashboard-components.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+// import Dashboard from '../../dashboard';
+import UserGuide from '../user-guide';
+import Sidebar from 'Components/dashboard/tutorial-tab/sidebar';
+
+jest.mock('Stores/connect', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect:
+        () =>
+        <T,>(Component: T) =>
+            Component,
+}));
+
+jest.mock('@deriv/bot-skeleton', () => ({
+    Blockly: jest.fn(),
+}));
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Tabs: jest.fn(() => true),
+    performSelfExclusionCheck: jest.fn(),
+}));
+
+describe('<Dashboard />', () => {
+    it('user guide button click', () => {
```

Comment:
@rupato-deriv probably the first test has no meaningful meaning and we do a similar test in the second test...
what is the difference between  test 1:'user guide button click' and test 2: on user guide button click it should render the tutorials tab ? Just that the user do click on the first test, but on the second test we check it again...

Reply:
@maryia-matskevich-deriv the first test case is a unit test case to check whether the button is clicked 
the second is an integration test case so we know the end result of the test case when we click the button the user should reach on. tutorials tab.

Reply:
@rupato-deriv please check my comment https://github.com/binary-com/deriv-app/pull/7903/files#r1154285442 🙂